### ISSUE TITLE
USER ISSUE: Candles loose fuel if moved

### ISSUE THREAD

**Issue by pynguin** [StrangeLoopGames/EcoIssues#5972]
**Version:** 0.7.2.3 beta

**Steps to Reproduce:**
place a candle
add one tallow
remove the candle with the hammer
place the candle again, and note that the tallow is gone.  this is the problem with coal in a stove as well.

**Expected behavior:**
should keep the candle fueled when moving it to new location	

**Actual behavior:**
loss of valuable fuel

**Comment by MarjieAVolk** [comment]
Yeah it's kind of a hard thing to rectify, since the only fuel you lose is the one that is partially consumed.  So you can't really get it back since it's partially consumed.  And it's also annoying to not allow you to pick the object up when it has partially consumed fuel.  So not sure how to fix it.

**Comment by SRMM** [comment]
Would it help to add items like candles with a "depletion bar" that displays the fuel left? (similar to tools)
They might be unable to stack (don't know how to handle that). When you remove one where fuel is empty they return to their normal stackable version.

**Comment by pynguin** [comment]
That is aligned with what I was thinking, so for candles and stoves and other objects that require fuel, essentially show them with a type of depletion bar, but maybe it is a different color to indicate the object is fine, but that is just needs fuel.  like a blue bar for example.  speaking of these items, I have not seen them pulling fuel from a linked inventory, not sure if I just missed it occurring of if there is a bug.  I can file a separate item for that though.
________________________________

**Comment by MarjieAVolk** [comment]
That sounds plausible, good idea.

@pynguin Fueled objects don't actually pull from linked inventories right now.  That is something we will likely add when I get around to https://github.com/StrangeLoopGames/EcoIssues/issues/2190 and https://github.com/StrangeLoopGames/EcoIssues/issues/2220

**Comment by metachronism** [comment]
another option is rolling this into the scrap system

**Comment by metachronism** [comment]
(assigning to craig as more of a design decision)
